template/main.js

```diff
@@ -617,8 +617,15 @@ function init($, _, locale, Handlebars, apiProject, apiData, Prism, sampleReques
     // and would make it jump to the wrong position or not jump at all.
     if (window.location.hash) {
         var id = decodeURI(window.location.hash);
-        if ($(id).length > 0)
-            $('html,body').animate({ scrollTop: parseInt($(id).offset().top) }, 0);
+        if(!!id) {
+            try {
+                // Prevent special characters from causing errors
```

Comment:
Add in the comment which characters you are replacing.

Reply:
Some characters, such as `/` , `Space` , `(` , `)` , `.`

Reply:
I mean add in the comment in the code.

Reply:
Okay, I done it